### ISSUE TITLE
NullPointerException in shallowClone().toString() due to lack of parent

### ISSUE THREAD

**Issue by wnm3** [jhy/jsoup#1410]
I am using 1.13.1 org.jsoup.jsoup and have code  like this:
```
String nodeHTML = ((Element) node).shallowClone().toString();
```
and get a NPE because Element.isInlineable  at 1619 calls parent().isBlock() but a shallowClone has no parent.

Suggest revising test to be && (parent()  == null || parent().isBlock()) rather than just parent().isBlock() since the latter causes NPE (unless no parents is not Inlineable...)

**Comment by gtaborcrossword** [comment]
Also `toString()` (internally `outerHtml()`) fails on this. So it seems to be a serious problem.

**Comment by jhy** [comment]
Thanks all. This was fixed with 77fcaf4dcdc8fd185e2f5d8eb6aa9c425a38bc9b. In particular, adding the nullability annotations created a warning for this issue.
